### PR TITLE
Disable Arm32 Builds

### PR DESCRIPTION
**Pull request by Craftzman7**
As requested in the [Discord](https://discord.com/channels/725836368059826228/725846784131203222/1241759253258047518) this PR disables Arm32 builds and removes mention of the build option in the README.

### CODE REVIEW

**Review by Bankst — APPROVED** [review]

**Comment by mcm001** [comment]
Closes #1324

**Review by mcm001 — APPROVED** [review]
🛥️

File: photon-server/src/main/java/org/photonvision/Main.java [hunk 1]
```diff
@@ -347,6 +347,27 @@ private static void addTestModeSources() {
...
+        logger.info(
+                "Starting PhotonVision version "
+                        + PhotonVersion.versionString
+                        + " on "
+                        + Platform.getPlatformName()
+                        + (Platform.isRaspberryPi() ? (" (Pi " + PiVersion.getPiVersion() + ")") : ""));
+
+        try {
+            if (!handleArgs(args)) {
+                System.exit(0);
```
Comment (mcm001) [inline photon-server/src/main/java/org/photonvision/Main.java:359]: Nit for future: why do we exit 0 on failure?

File: photon-server/src/main/java/org/photonvision/Main.java [hunk 2]
```diff
@@ -347,6 +347,27 @@ private static void addTestModeSources() {
...
+            }
+        } catch (ParseException e) {
+            logger.error("Failed to parse command-line options!", e);
+        }
+
+        // We don't want to trigger an exit in test mode. This is specifically for MacOS.
+        if (!Platform.isSupported() && !isSmoketest) {
```
Comment (mcm001) [inline photon-server/src/main/java/org/photonvision/Main.java:366]: Smoke test and test mode are two different things. We want this check to be skipped on both.

File: photon-server/src/main/java/org/photonvision/Main.java (hunk 1 above)
Comment (mcm001) [reply photon-server/src/main/java/org/photonvision/Main.java:359]: If you wanna exit 1 or something with a error print now, sure

File: photon-server/src/main/java/org/photonvision/Main.java [hunk 3]
```diff
@@ -347,6 +347,27 @@ private static void addTestModeSources() {
...
+        try {
+            if (!handleArgs(args)) {
+                System.exit(0);
+            }
+        } catch (ParseException e) {
+            logger.error("Failed to parse command-line options!", e);
+        }
+
+        // We don't want to trigger an exit in test mode or smoke test. This is specifically for MacOS.
+        if (!Platform.isSupported() && (!isSmoketest || !isTestMode)) {
```
Comment (mcm001) [inline photon-server/src/main/java/org/photonvision/Main.java:366]: I would double check your Boolean logic here...

**Review by mcm001 — CHANGES_REQUESTED** [review]

**Review by mcm001 — APPROVED** [review]

**Pull request merged**